lib/helper/Playwright.js

```diff
@@ -346,7 +346,10 @@ class Playwright extends Helper {
 
     if (currentUrl.startsWith('http')) {
       await this.executeScript('localStorage.clear();').catch((err) => {
-        if (!(err.message.indexOf("Storage is disabled inside 'data:' URLs.") > -1)) throw err;
+        if (!(err.message.indexOf("Local storage is disabled inside 'data:' URLs.") > -1)) throw err;
```

Comment:
Not sure on this, we already checked that url starts with http? 
There is a possibility that we are on data: 

What if other error happens, should we throw error here?

Reply:
Yes, we should.

Reply:
And this line should throw any Error, except for Storage.